TUTORIAL.md

```diff
@@ -434,6 +436,20 @@ end
 
 For an argument, each function will be tried in order. The first function that doesn't raise a `ContractError` will be used. So in this case, if x == 1, the first function will be used. For all other values, the second function will be used.
 
+This allows you write methods more declaratively, rather than using conditionals. This feature is not only useful for recursion; you can use it to keep parallel use cases separate:
```

Comment:
it is just my suggestion to your comment https://github.com/egonSchiele/contracts.ruby/issues/79#issuecomment-73114220